corehq/apps/hqwebapp/views.py

```diff
@@ -507,17 +506,6 @@ def dropbox_upload(request, download_id):
     return HttpResponseRedirect(request.META.get('HTTP_REFERER', '/'))
 
 
-# TODO jschweers: Remove this in Q2 2018 if it hasn't been triggered,
-# change initial_page_data.js to throw an error
```

Comment:
Does this need to throw an error, or is it taken care of?

Reply:
The premise isn't correct/relevant anymore. I had wanted to assert that initial page data isn't accessed until the document is fully loaded, but then wrote [some code](https://github.com/dimagi/commcare-hq/pull/18899) that does deliberately access ipd before load is complete. I thought that code would be temporary but now think it's more likely to be permanent (as much as any code is permanent).

Really I did this PR because I was making a plan to split up hqwebapp/views.py, this was a view without an obvious home, and then I realized it hasn't been called in a year and deleting it seemed more constructive than dithering about where to put it.